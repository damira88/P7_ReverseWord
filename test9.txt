Challenges-are-what-make-life-interesting-and-overcoming-them-is-what-makes-life-meaningful.