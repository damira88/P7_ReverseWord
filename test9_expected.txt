Output:segnellahC-era-tahw-ekam-efil-gnitseretni-dna-gnimocrevo-meht-si-tahw-sekam-efil-.lufgninaem